src/pages/LeadershipPage.jsx

```diff
@@ -3,26 +3,48 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import LinearProgress from '@material-ui/core/LinearProgress';
 
 import ContentCard from '../components/contentCard/ContentCard';
 import { UPDATE_CURRENT_PAGE_TITLE } from '../store/constants';
+import { getLeadershipData } from '../store/actions';
 import NotFound from './errors/notFound';
 
 const propTypes = {
   setHeaderTitle: PropTypes.func.isRequired,
-  leadershipData: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  leadershipData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      position: PropTypes.oneOf(
+        ["President", "Vice President", "Treasurer", "Secretary", "Public Relations", "Mechanical Lead", "Website Manager"]
+      ).isRequired,
+      bio: PropTypes.string.isRequired,
+    }
+  )).isRequired,
+  fetchLeadershipData: PropTypes.func.isRequired,
 };
 
 const PAGE_TITLE = 'Leadership';
 
-const LeadershipPage = ({ setHeaderTitle, leadershipData }) => {
+const LeadershipPage = ({ setHeaderTitle, leadershipData, fetchLeadershipData }) => {
+  const [isLoading, setIsLoading] = React.useState(true);
```

Comment:
There should also be an error state.

Reply:
added here https://github.com/MST-Game-Development/website/pull/22/commits/e404a22a38c7e06223fc6ea9fce9e4592fa59b25